src/_pytest/capture.py

```diff
@@ -618,7 +618,7 @@ class FDCapture(FDCaptureBinary):
     snap() produces text
     """
 
-    EMPTY_BUFFER = str()
+    EMPTY_BUFFER = str()  # type: ignore
```

Comment:
what's the type error here?  this should also probably just be `''` instead of `str()` -- maybe I'll teach `pyupgrade` to do that :thinking:  https://github.com/asottile/pyupgrade/issues/187

Reply:
There superclass `FDCaptureBinary` sets this to `bytes`. I didn't try to understand too closely what this does TBH.

Reply:
`FDCapture` should probably be generic